bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/listeners/SetupListener.java

```diff
@@ -56,7 +56,7 @@ private boolean compatible(Alloc pred, Alloc succ) {
 		return 
 			pred != null &&
 			pred.getObs().getInstrumentString().equals(succ.getObs().getInstrumentString()) &&
-			pred.getObs().getCoords().equals(succ.getObs().getCoords());
+			pred.getObs().getCoords(pred.getEnd()).equals(succ.getObs().getCoords((succ.getEnd())));
```

Comment:
For a non-sidereal target, it seems unlikely that this will ever return `true`.  I think what we're trying to get at here is whether we're still observing the same target.  Since we have no unique ids for targets, we're doing this by checking whether the previous and next observation coordinates are in the same place.  Since targets are moving, to make this check meaningful I believe we need to compare them at the same time.  It's technically possible that two distinct targets could be in exactly the same place at one time and then move apart, but exceedingly unlikely.  So I think pick a time, either `pred.getEnd` or `succ.getStart` and use it for both coordinate lookups.  Alternatively, we could do two comparisons, one of both coordinates at `pred.getEnd` and another of both coordinates at `succ.getStart` but I don't think it's necessary and at any rate still doesn't guarantee correctness.